lms/djangoapps/teams/static/teams/js/spec/views/teams_tab_spec.js

```diff
@@ -137,5 +137,92 @@ define([
                 })).toBe(true);
             });
         });
+
+        describe('Search', function () {
+            var verifyTeamsRequest = function(requests, options) {
+                AjaxHelpers.expectRequestURL(requests, 'api/teams/',
+                    _.extend(
+                        {
+                            topic_id: 'test_topic',
+                            expand: 'user',
+                            course_id: 'test/course/id',
+                            order_by: '',
+                            page: '1',
+                            page_size: '10',
+                            text_search: ''
+                        },
+                        options
+                    ));
+            };
+
+            it('can search teams', function () {
+                var requests = AjaxHelpers.requests(this),
+                    teamsTabView = createTeamsTabView();
+                teamsTabView.browseTopic('test_topic');
+                verifyTeamsRequest(requests, {
+                    order_by: 'name',
+                    text_search: ''
+                });
+                AjaxHelpers.respondWithJson(requests, {});
+                teamsTabView.$('.search-field').val('foo');
+                teamsTabView.$('.action-search').click();
+                verifyTeamsRequest(requests, {
+                    order_by: '',
+                    text_search: 'foo'
+                });
+                AjaxHelpers.respondWithJson(requests, {});
+                expect(teamsTabView.$('.page-title').text()).toBe('Search Topic: test topic');
+                expect(teamsTabView.$('.page-description').text()).toBe('Showing results for "foo"');
+            });
+
+            it('can clear a search', function () {
```

Comment:
Personally, I would just combine this with the previous test point so you can be sure that the setup was correct (before the clear).